README.md

```diff
@@ -1,9 +1,25 @@
-# openinput-cm3
+# openinput-mouse
```

Comment:
I renamed the repo to just `openmouse`, can you match?

There is no essential difference in firmware between mice and keyboards. The only thing is that mice will have an extra component, a sensor. We will support macros in mice, which forces us to have the HID Keyboard Usage Page anyway, there is no reason to maintain a different firmware just for keyboards.

Reply:
Will do.